webtau-http/src/main/java/com/twosigma/webtau/http/HttpDocumentation.java

```diff
@@ -52,6 +55,11 @@ public void capture() {
             capturePaths();
         }
 
+        private void captureUrl() {
+            FileUtils.writeTextContent(path.resolve("request.url.path.txt"), UrlUtils.extractPath(lastValidationResult.getFullUrl()));
```

Comment:
I wonder if we should capture the URL user provided in a test? Does `extractPath` keeps `query` part of the url?

Reply:
That's a great point and no, it won't.  I had considered that initially but the result object the capture looks at does not have the user specified url.  I'll see what it'll take to have it included in there but I agree the distinction should ideally be user specified part vs. full url rather than path vs. url.

One interesting question is what will happen if someone specifies query params but not in the url part.  I guess I'll test and find out 😄